Source/WebCore/platform/graphics/texmap/TextureMapper.h

```diff
@@ -50,23 +50,18 @@ enum class TextureMapperFlags : uint16_t;
 class TextureMapper {
     WTF_MAKE_FAST_ALLOCATED;
 public:
-    enum PaintFlag {
-        PaintingMirrored = 1 << 0,
-    };
-
     enum WrapMode {
         StretchWrap,
         RepeatWrap
     };
 
-    typedef unsigned PaintFlags;
-
     WEBCORE_EXPORT static std::unique_ptr<TextureMapper> create();
 
     TextureMapper();
     WEBCORE_EXPORT ~TextureMapper();
 
     enum class AllEdgesExposed : bool { No, Yes };
+    enum class ShouldPaintMirrored : bool { No, Yes };
```

Comment:
`ShouldPaintMirrored` seems to be less informative to me. How about `InvertedYAxis` or `YAxisInverted`?

Reply:
Good point, GraphicsContextGL uses `enum class FlipY : bool { No, Yes };`. Flip y is commonly used in other projects too.